File: cpp/program61.cpp
#include<iostream>
using namespace std;

class Pattern
{
    public:
        int iNo;

        Pattern(int X)
        {
            iNo = X;
        }

        void Display()
        {
            int iCnt = 0;

            for(iCnt = 1; iCnt <= iNo; iCnt++)
            {
                cout<<iCnt<<"\t";
            }
            for(iCnt = iNo-1; iCnt >= 1; iCnt--)
            {
                cout<<iCnt<<"\t";
            }
        }
};

int main()
{
    int iValue = 0;

    cout<<"Enter frequency\n";
    cin>>iValue;

    Pattern *obj = new Pattern(iValue);
    obj->Display();

    return 0;
}

File: java/program61.java
import java.util.*;

class program61
{
    public static void Display(int no)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= no; iCnt++)
        {
            System.out.print(iCnt+"\t");
        }
        for(iCnt = no-1; iCnt >= 1; iCnt--)
        {
            System.out.print(iCnt+"\t");
        }
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter frequency");
        int value = sobj.nextInt();

        Display(value);
    }
}